app/models/form/sales/questions/nationality_all.rb

```diff
@@ -0,0 +1,221 @@
+class Form::Sales::Questions::NationalityAll < ::Form::Question
+  def initialize(id, hsh, page, buyer_index)
+    super(id, hsh, page)
+    @check_answer_label = "Buyer #{buyer_index}’s nationality"
+    @header = "Enter a nationality"
+    @type = "select"
+    @answer_options = ANSWER_OPTIONS
+    @check_answers_card_number = buyer_index
+    @question_number = buyer_index == 1 ? 24 : 32
+  end
+
+  ANSWER_OPTIONS = {
+    "" => "Select an option",
```

Comment:
can we store this list somewhere central?

Reply:
I thought about that, but I feel like it's very question specific so putting it into a constant might be inviting for it to get reused by other things where it shouldn't
let me know if you feel strongly about this and I'm missing any non obvious benefits

Reply:
I was wondering about this - the main thing I can think of is not having to update it in two places (lettings and sales) if it changes. But I imagine it doesn't change very often, save maybe adding some synonyms. 

That said, are there likely to be things that would want to reuse it but shouldn't? I might be missing something here, but it does seem like anything using a nationality list should want the same list (and also that there wouldn't be other places using a nationality list).

Reply:
Yeah, it's unlikely we'd want it somewhere els. I think I was thinking about the local authorities and not countries and was worried about the structure changing as opposed to the content, cause it's quite question specific. I've extracted it now tho, let me know if it looks better

Reply:
Yeah, I think that works.